examples/gocean/eg3/Makefile

```diff
@@ -1,7 +1,7 @@
 # -----------------------------------------------------------------------------
 # BSD 3-Clause License
 #
-# Copyright (c) 2020, Science and Technology Facilities Council.
+# Copyright (c) 2021, Science and Technology Facilities Council.
```

Comment:
Strictly 2020-2021 (because this file was already present I think?)

Reply:
Ups.. good catch